packages/babel-plugin/src/visitors/stylex-create/parse-stylex-create-arg.js

```diff
@@ -72,14 +73,15 @@ export function evaluateStyleXCreateArg(
     const allParams: Array<
       NodePath<t.Identifier | t.SpreadElement | t.Pattern>,
     > = fnPath.get('params');
+
+    validateDynamicStyleParams(allParams);
```

Comment:
hmm. this works fine when I just run transformation, but I tried to get these messages thrown via the webpack plugin and they get swallowed. I wonder if we should just propogate the error message to the `transformStyleXCreate` function and then throw it from there

Reply:
The webpack plugin tends to swallow errors, let's focus on making the errors work when Babel is used directly in this PR. Making the bundler plugins handle those errors correctly is a separate task.